### PR TITLE
Fix broken package builds

### PR DESCRIPTION
**Pull request by ghost**
PR #134 should be rebased on top of these commits.

### CODE REVIEW

**Comment by ghost** [comment]
start build

File: lib/package.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def download_files(self, recurse=True):
         """
         # Download all package sources
         repositories_path = CONF.get('default').get('repositories_path')
-        download_f = partial(package_source.download, dest=repositories_path)
+        download_f = partial(package_source.download, directory=repositories_path)
```
Comment (farosas) [inline lib/package.py:102]: This change was already merged by #135.

**Comment by ghost** [comment]
start build

**Comment by farosas** [comment]
start build

**Review by farosas — APPROVED** [review]

**Pull request merged**